WooCommerce/Classes/ViewRelated/Products/ProductSelector/ProductSelectorViewModel.swift

```diff
@@ -32,12 +32,12 @@ final class ProductSelectorViewModel: ObservableObject {
     /// View model for the filter list.
     ///
     var filterListViewModel: FilterProductListViewModel {
-        FilterProductListViewModel(filters: filters, siteID: siteID)
+        FilterProductListViewModel(filters: filters.value, siteID: siteID)
     }
 
-    /// Selected filter for the product list
+    /// Selected filters for the product list
     ///
-    @Published var filters = FilterProductListViewModel.Filters()
+    var filters = CurrentValueSubject<FilterProductListViewModel.Filters, Never>(.init())
```

Comment:
No reason, I'm just not acquainted to use Combine and I though "if it changes, it is a var" 😅  

I just made it `let` to avoid complexity for the moment, since we don't need to re-assign the subject. Change on 55600d6967afe03eb0dd7de18be295244e572891